src/main.cpp

```diff
@@ -569,6 +616,9 @@ auto main(int _argc, char* _argv[]) -> int
         const auto eviction_check_interval = http_server_config.at(json::json_pointer{"/authentication/eviction_check_interval_in_seconds"}).get<int>();
         bearer_token_eviction_manager eviction_mgr{ioc, std::chrono::seconds{eviction_check_interval}};
 
+		// TODO: Don't lean on BT timer
+		oidc_state_eviction_manager oidc_cleanup{ioc, std::chrono::seconds{eviction_check_interval}};
```

Comment:
Another config option for state eviction checks?

Reply:
I think that depends on answers to the other questions.

Reply:
Class removed.